test/kitchen/kitchen.dokken.yml

```diff
@@ -34,6 +34,11 @@ platforms:
       volumes:
         - /sys/fs/cgroup:/sys/fs/cgroup:ro # required by systemd
 
+  - name: centos-6
```

Comment:
CentOS 6 EOL is 11/30/2020 -- I'm not sure adding this to the test matrix would be beneficial at this juncture.

Reply:
I wanted to show that the correction for sysV systems did indeed work. I'll reverse these changes.